tutorial/openmp.md

```diff
@@ -0,0 +1,571 @@
+# Pyccel OpenMP usage
+
+## Using the Runtime Library Routines
+
+OpenMP Runtime Library Routines for Pyccel work by importing the OpenMP routine needed from the Pyccel stdlib:
+
+```python
+from pyccel.stdlib.internal.openmp import omp_set_num_threads
+```
+
+### Example
+
+The following example show how ``` omp_set_num_threads ``` is used to set the number of threads to ``` 4 threads ``` and how ``` omp_get_num_threads ``` is used to get the number of thread in the current team within a parallel region, ``` omp_get_num_threads ``` will return ``` 4 threads ```.
+
+```python
+from pyccel.decorators import types
+
+@types('int')
+def get_num_threads(n):
+    from pyccel.stdlib.internal.openmp import omp_set_num_threads, omp_get_num_threads, omp_get_thread_num
+    omp_set_num_threads(n)
+    #$ omp parallel
+    print("hello from thread number:", omp_get_thread_num())
+    result = omp_get_num_threads()
+    #$ omp end parallel
+    return result
+x = get_num_threads(4)
+print(x)
```

Comment:
Since the variable `result` first appeared in the parallel region, one could take it as if it's private to each thread in the parallel region. Don't you think we should make it clear that every variable is shared unless it's explicitly privatized using clauses?